Firmware/lcd.cpp

```diff
@@ -95,21 +95,6 @@ static const CustomCharacter Font[] PROGMEM = {
 #define CUSTOM_CHARACTERS_CNT (sizeof(Font) / sizeof(Font[0]))
 
 static void lcd_display(void);
-
-#if 0
```

Comment:
since we are removing these, might as well remove the VT100 code as we don't need it anymore and takes tons of flash when enabled.
Still thinking what to do about the 8bit lcd code. I think I'll keep that one as it's not too intrusive.

Reply:
Done 👍